docs/governance.rst

```diff
@@ -0,0 +1,244 @@
+   Issue.
+2. Every modification of the specification (including a correction of a typo,
+   adding a new Contributor, an extension adding support for a new data type,
+   or others) or proposal to release a new version needs to be done via a
+   Pull Request (PR) to the Repository.
+3. Anyone can open an Issue or a PR (this action is not limited to Contributors).
+4. A PR is eligible to be merged if and only if these conditions are met:
+
+   a) The PR features at least two `Reviews that Approve <https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews>`_
+      the PR of which neither is the author of the PR. 
+      The reviews should be made after the last commit in the PR (equivalent to 
+      `Stale review dismissal <https://help.github.com/articles/enabling-required-reviews-for-pull-requests/>`_
+      option on GitHub). If a second review requests minor changes after
+      another reviewer approved the PR, the first review does not need
+      to re-review.
+   b) Does not feature any `Reviews that Request changes <https://help.github.com/articles/about-required-reviews-for-pull-requests/>`_.
+      That is, if someone asked for changes, the PR should not be merged just
+      because two other people approve it.
+   c) Is not a Draft PR. That is the PR author says it is ready for review.
+   d) Passes all automated tests.
+   e) Is not proposing a new release
+   f) The steering committee has not added extra restrictions. For example, if
+      a PR is a non-trival change, the steering committee can create a system
+      to get feedback from more than just two reviewers before merging.
+5. After consultation with contributors, the steering committee can decide
+   to merge any PR - even if it's not eligible to merge according to Rule 4.
+6. Anyone can Review a PR and request changes. If a community
+   member Requests changes they need to provide an explanation regarding what
+   changes should be made and justification of their importance. Reviews
+   requesting changes can also be used to request more time to review a PR.
+7. A reviewer who Requested changes can dismiss their own review, if
+   they decide their requested changes are no longer necessary, or approve
+   changes that address the issue underlying their change request.
+8. If the author of a PR and a reviewer who requests changes cannot find a
+   solution that would lead to: (1) The author closing the PR without merging
+   (2) The reviewer accepting requested changes or (3) The dismissing their
+   review, so that the PR can be approved and merged, then the disagreement
+   will be resolved with a vote.
+9. Rules governing voting:
+
+   a) A vote can be triggered by any Maintainer, but only after 5 working days
+      from the time a Review Requesting Changes. If a PR has a disagreement
+      that required a vote, there must be at least 15 days from the conclusion
```

Comment:
Hm, on further reflection I'm not sure if I agree. This could easily let things drag out much more than they otherwise would. I'd be curious to hear what others think.

Reply:
I didn't change this from @tsalo's original draft. I suspect this makes sense for a larger project. I assume the spirit of this rule is to not have a PR jump from one contentious vote to another. I hope there are simpler solutions. For example, we allow the steering committee to change the rules so, if something is contentious, then can set up a system for resolving the contention rather than jumping from vote to vote.

Reply:
There was a lot of discussion around this point in BIDS, and the thought was that a longer time period was necessary given the global nature of the project and the conflicting time commitments that so many members have.

That being said, we're at the point in BIDS where a vote basically never happens -- and I think that's (largely) a good thing. Votes are usually a sign that something in scope, etc, are unclear, if no one can agree and there's a need to poll the community. It also exhausts your contributors's time if they're being called for votes too often, whch means they have less bandwidth to actually contribute to the project.

I'm fine to copy the rules as-is with this understanding that votes are really the fall-back. If we're hitting them often, I think we need to take a deep look at where the confusion is arising and have a very clear discussion / decision on that.